x-pack/platform/packages/shared/kbn-elastic-assistant-common/impl/schemas/conversations/common_attributes.schema.yaml

```diff
@@ -332,9 +332,6 @@ components:
         apiConfig:
           $ref: '#/components/schemas/ApiConfig'
           description: LLM API configuration.
-        isDefault:
-          description: Is default conversation.
-          type: boolean
```

Comment:
WRT API breaking changes since the [Conversations API is already public](https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-createconversation) -- this is only an issue if it's still included as request validation with fail, correct? This seems fine given adoption and functionality here, so if anything maybe we just make a note to mention in the release notes?